### ISSUE TITLE
Could not load file or assembly 'System.Buffers, Version=4.0.2.0

### ISSUE THREAD

**Issue by yannisalexiou** [alexalok/dotAPNS#81]
## Expected Behavior
To send a notification using Token-based connection in Azure Environment.

## Actual Behavior
In localhost environment under the IIS everything working perfect.
But in Azure i receive the below error under `catch (HttpRequestException ex)`

What I miss? Why the HTTPRequest searches for System.Buffers, Version=4.0.2.0 while I have installed v4.5.1?

```c#
System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> 
System.IO.FileLoadException: Could not load file or assembly 'System.Buffers, Version=4.0.2.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)   
at System.Net.Http.WinHttpResponseParser.CreateResponseMessage(WinHttpRequestState state, DecompressionMethods manuallyProcessedDecompressionMethods)   
at System.Net.Http.WinHttpHandler.<StartRequestAsync>d__105.MoveNext()   
--- End of inner exception stack trace ---   
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)   
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)   
at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)   
at dotAPNS.ApnsClient.<SendAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown --- 
```

## Specifications

  - Version: dotAPNS v 3.4.0, System.Buffers v4.5.1 is installed
  - Platform: .NET Framework 4.8, Azure
  - Subsystem: Visual Studio Code 2019

**Comment by yannisalexiou** [comment]
**TL;DR; Updating System.Net.Http.WinHttpHandler to latest version (v.5) solves the issue.**

The `WinHttpHandler` uses `System.Buffers`.
It seems `System.Buffers` was in a newer version not supported by `WinHttpHandler`.

I also rebuild the reassembly redirects in web.config just to be sure.
From Package Manager console: 
```
Get-Project –All | Add-BindingRedirect
```

**Comment by alexalok** [comment]
Hi @yannisalexiou,

glad you've managed to resolve the issue! Thank you for sharing with others as well.
